### ISSUE TITLE
Unfounded parse errors in Haskell kata creation

### ISSUE THREAD

**Issue by geppettodivacin** [Codewars/codewars.com#310]
I am translating the kata "Write your own multiplication function" to Haskell. I have the tests and code written, but it refuses to compile on the basis of parse errors that shouldn't exist. Although I've compiled and run identical code on my own machine, the compiler gives me errors such as the following:

```
 /tmp/haskell116419-15-11gob3u/MultiplicationFunction/Preload.hs:7:21: Parse error: naked expression at top level
```

The code there is a type signature, and in fact the character at the column given is in the middle of a word:

``` haskell
number :: Int -> Number
```

I had a greater number of errors before that had a slightly different form:

```
/tmp/haskell116419-23-ie7j5l/Haskell/Codewars/MultiplicationFunction.hs:5:24: parse error on input `&#'
```

Those two characters never appeared in the source.

This is the first kata that I have edited or created, so I don't know if this is a more widespread problem with Preloaded functions in Haskell or if it is just for translations. Has anyone else had this type of problem? And is there any idea where this could be coming from?

**Comment by geppettodivacin** [comment]
I tried putting the same code in a new kata and a new kumite. The code worked just fine in the kata, but I received the same parse errors when attempting to work with the kumite. I published the kumite as Test Haskell Kumite.

**Comment by geppettodivacin** [comment]
Interestingly, the problems go away when using publish as opposed to Run.

**Comment by geppettodivacin** [comment]
The translation was published just fine. Nevertheless, the parser used when the "Run" button is pressed in a kumite is still broken. The parser works just fine with "Publish."

**Comment by bkaestner** [comment]
Possibly related to #248.

**Comment by geppettodivacin** [comment]
I noticed the same thing as #248, although they seem unrelated. With the Run button, the tests were applied to the old code, but the parse errors reflected code that never existed. However, it was consistent based on the code that I gave it: given the same code (and a refresh), I received the same error, and moving the line that the error was on (with blank lines) produced the same error on the new line.
